exe/delayed_job

```diff
@@ -2,4 +2,4 @@
 set -e
 
 cd /app
-exec chpst -u app env QUEUE=coldp_export,basic_nomenclature_export /usr/bin/bundle exec rails jobs:work
+exec chpst -u app env QUEUE=coldp_export,basic_nomenclature_export,dwc_occurrence_index,import_dataset_import,import_dataset_stage /usr/bin/bundle exec rails jobs:work
```

Comment:
Might need a cleaner way to list all these, or some standardization at some point.

Reply:
After merge we need to decide whether we keep limiting which queues to run or just process all queues.